src/main/java/com/benko/exercise/sort/Sorter.java

```diff
@@ -0,0 +1,8 @@
+package com.benko.exercise.sort;
+
+/**
+ * Created by Robert on 2019-01-06.
```

Comment:
Wiem że edytor ustawia te komentarze o tym kto stworzył ale osobiście to ich nie lubię. Taka historia jest w gitcie. Jeśli javadoc ma być to niech ma istotne informacje.
Proszę usun ten komentarz